src/main/java/io/jenkins/plugins/constant/Constant.java

```diff
@@ -6,7 +6,7 @@ public class Constant {
     public static final String UNIQUE_FILE_NAME = "delphix-VDB-";
     public static final String PROPERTIES = ".properties";
     public static final String FILE_NAME = "delphix-VDB";
-    public static final String USER_AGENT = "Jenkins-3.1.1";
+    public static final String USER_AGENT = "Jenkins-3.2.1";
```

Comment:
In the `pom.xml` the version says `3.2.0`. 

Anyway we can dynamically resolve the version so we don't have this issue?

Reply:
I don't think the `pom.xml` issue is addressed. 

![image](https://github.com/jenkinsci/delphix-plugin/assets/114104601/00126336-44d3-4bcd-9632-6a779c13f466)

Reply:
I didn't realize the jenkins bot updates this value.